win32ss/drivers/videoprt/int10.c

```diff
@@ -298,7 +357,11 @@ IntInt10WriteMemory(
     INFO_(VIDEOPRT, "- Length: %x\n", Length);
 
     IntAttachToCSRSS(&CallingProcess, &ApcState);
+    if (IsLowV86Mem(Seg, Off))
```

Comment:
There appears to be other places in the code:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=IntAttachToCSRSS
where this unprotection/protection whenever IntAttachToCSRSS is called, may be needed.

Reply:
The only place left is here, and the accessed address is above the protected range.
https://git.reactos.org/?p=reactos.git;a=blob;f=win32ss/drivers/videoprt/videoprt.c;hb=73fd52a164ac9f2cc50462295ec2f5eb23450a25#l1030

Reply:
Thanks!